src/confcom/azext_confcom/tests/latest/test_confcom_kata.py

```diff
@@ -49,22 +48,37 @@ def test_invalid_input_path(self):
 
     def test_invalid_config_map_path(self):
         filename = "pod.yaml"
-        with open(filename, "w") as f:
-            f.write(KataPolicyGen.pod_string)
-        with self.assertRaises(SystemExit) as wrapped_exit:
-            katapolicygen_confcom(
-                filename, "fakepath/configmap.yaml",
-            )
-        os.remove(filename)
+        try:
+            with open(filename, "w") as f:
+                f.write(KataPolicyGen.pod_string)
+            with self.assertRaises(SystemExit) as wrapped_exit:
+                katapolicygen_confcom(
+                    filename, "fakepath/configmap.yaml",
+                )
+        finally:
+            os.remove(filename)
```

Comment:
nit: maybe there should be a check if the file exists in the `finally` block, just in case the write step fails?